sov-modules/sov-modules-api/src/default_signature.rs

```diff
@@ -1,50 +1,92 @@
 use crate::{SigVerificationError, Signature};
 use borsh::{BorshDeserialize, BorshSerialize};
+use ed25519_dalek::ed25519::signature::Signature as DalekSignatureTrait;
+use ed25519_dalek::Verifier;
+use ed25519_dalek::{Keypair, PublicKey as DalekPublicKey, Signature as DalekSignature, Signer};
 
-// TODO: https://github.com/Sovereign-Labs/sovereign/issues/253
-#[derive(PartialEq, Eq, Clone, BorshDeserialize, BorshSerialize, Debug)]
-pub struct DefaultPublicKey {
-    pub(crate) pub_key: Vec<u8>,
+use rand::{CryptoRng, RngCore};
+
+pub struct DefaultPrivateKey {
+    key_pair: Keypair,
 }
 
-impl DefaultPublicKey {
-    pub fn new(pub_key: Vec<u8>) -> Self {
-        Self { pub_key }
+impl DefaultPrivateKey {
```

Comment:
We won't ever sign in zk, so let's feature gate this.

Reply:
We decided on daily that the signature scheme will be implemented in the following PR.